src/main/resources/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl/credentials.jelly

```diff
@@ -32,6 +32,9 @@
   <f:entry title="${%Secret Access Key}" field="secretKey">
     <f:password/>
   </f:entry>
+  <f:entry title="${%AWS Region}" field="awsRegion">
+    <f:textbox default="us-east-1" />
```

Comment:
Better to put the default in the field, that way you have the default both when using the object from Jelly and programatically.

Reply:
but setting this` <f:textbox default="us-east-1" />` is not already setting the UI with this default value?

Reply:
Yes, but this object can be used both from the UI and programmatically. Given that you have the default above you may remove it here.

Reply:
removed